Sources/_CryptoExtras/RSA/RSA_boring.swift

```diff
@@ -465,6 +469,22 @@ extension BoringSSLRSAPublicKey {
         deinit {
             CCryptoBoringSSL_EVP_PKEY_free(self.pointer)
         }
+
+        fileprivate func getKeyPrimitives() throws -> (n: Data, e: Data) {
+            let key = CCryptoBoringSSL_EVP_PKEY_get0_RSA(self.pointer)
+
+            func getPrimitive(_ getPointer: (OpaquePointer?) -> UnsafePointer<BIGNUM>?) throws -> Data {
+                let ptr = getPointer(key)
+                let size = Int(CCryptoBoringSSL_BN_num_bytes(ptr))
+                var buffer = [UInt8](repeating: 0, count: size)
```

Comment:
I don't think we need to involve a poor innocent temporary Array here: we may as well compute this directly on `Data`.